library/src/main/java/com/ricohapi/mstorage/MediaStorage.java

```diff
@@ -256,7 +259,39 @@ public void addMeta(String mediaId, Map<String, String> userMeta, CompletionHand
     }
 
     public void removeMeta(String mediaId, String key, CompletionHandler<Object> handler) {
-        // TODO
+        try {
+            if (accessToken == null) {
+                throw new RicohAPIException(0, "wrong usage: use the connect method to get an access token.");
+            }
+            RicohAPIRequest request;
+            if (key == null) {
```

Comment:
I don't think this needs to fix now.
Because `mediaId` is used to only combine to URL, even `mediaId` was `null`, just the generated URL is `.../media/null/...` .

On the other hand, `key` is used as parameter of `Pattern#matcher` method, it will throw `NullPointerException` when `key` is `null`. 
For avoiding this, it needs to check whether `key` is `null`.

Issue of checking `mediaId` is same for many other methods, so I created other ticket : [NECO-179](https://pflabo.atlassian.net/browse/NECO-179)